CloudWorkshop_DIPC_100.md

```diff
@@ -21,7 +21,7 @@ Your will need:
 ## Logging Into Oracle Cloud Instance
 
 1. In your web browser, navigate to cloud.oracle.com, then click Sign in.
-2. Provide the cloud account: oscnas001 then <Enter>
+2. Provide the cloud account: for example XXXXXXXXX then <Enter>
```

Comment:
Since we are assuming they are doing this in the workshop I would just say for example oscnas001 because saying "for example XXXXXXXXX" does NOT really help.